daily_milestone.go

```diff
@@ -1,29 +1,100 @@
 package main
 
-import "flag"
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"strings"
+)
 
-func get_project_id(base_url string, token string, projectname string, namespace string) string {
-	headers := map[string]string{"PrivateToken": token}
-	url := "https://" + base_url + "/projects"
+var logger *log.Logger
+
+//LoggerSetup Initialization
+func LoggerSetup(info io.Writer) {
+	logger = log.New(info, "INFO: ", log.Ldate|log.Ltime|log.Lshortfile)
+}
+
+func getProjectID(BaseURL string, Token string, Projectname string, Namespace string) string {
+	urls := "https://" + BaseURL + "/projects"
 	page := 1
+	headers := url.Values{}
+	headers.Set("PrivateToken", Token)
+	strPage := strconv.Itoa(page)
+	s := []string{urls, "?page=", strPage}
+	completeURL := strings.Join(s, "")
 	for {
+		resp, err := http.PostForm(completeURL, headers)
```

Comment:
PostForm uses the second argument (here: headers) to send specific formdata encoded into values.
Here we should probably not use PostForm, but a general post request.